src/pages/home/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from 'react';
+import { Navigate } from 'react-router-dom';
+import Navbar from '../../components/Navbar';
+import { getUserInfo } from '../../util/getUser';
+import { isUserLogged } from '../../util/loginUser';
+import './home.css';
+import UserPosts from './userPosts';
+interface ILogged {
+  isLogged: boolean;
+  userID: number;
+}
+
+const Home = () => {
+  const { isLogged, userID } = isUserLogged() as ILogged;
+  return (
+    <div className='Home'>
+      {!isLogged && <Navigate to='/login' replace={true} />}

```

Comment:
A better approach is to wrap your routes where you need to be logged in in a guard. Usually you'll have an `Authenticated` guard that you wrap your components with in the route file:

https://blog.netcetera.com/how-to-create-guarded-routes-for-your-react-app-d2fe7c7b6122

Reply:
Thank you i will check this doc, and i have a question, do you recommend me to use this Authenticated guard or protected routes with react router v6 ?